### PR TITLE
Add certificate field in the custom asset section of the demo.

### LINKED ISSUE #1135: Add certificate field to custom asset section in demo app
The demo app should include a 'Certificate URL' field in the custom asset section. This would be helpful for testing:
- Fairplay content
- Widevine content where the license server isn't configured to handle certificate requests

I can make a PR for this.

### PR DESCRIPTION
**Pull request by chrisfillmore**
Resolves #1135

### CODE REVIEW

File: demo/asset_section.js [hunk 1]
```diff
@@ -129,6 +131,45 @@ shakaDemo.onAssetKeyUp_ = function(event) {
 };
 
 
+/**
+ * @param {!string} uri
+ * @return {!Promise}
```
Comment (TheModMaker) [inline demo/asset_section.js]: You can be more specific on the return value.  `{!Promise.<ArrayBuffer>}`.

File: demo/index.html [hunk 2]
```diff
@@ -110,6 +110,10 @@ <h1>Shaka Player <span id="version"></span></h1>
             <label for="licenseServerInput">Custom license server:</label>
             <input id="licenseServerInput" type="text" class="flex-grow">
           </div>
+          <div class="flex">
+            <label for="certificateInput">Custom certificate:</label>
```
Comment (TheModMaker) [inline demo/index.html]: I think `Custom license certificate URL` would be better.

File: demo/asset_section.js [hunk 3]
```diff
@@ -238,8 +281,17 @@ shakaDemo.load = function() {
   // Revert to default poster while we load.
   shakaDemo.localVideo_.poster = shakaDemo.mainPoster_;
 
-  // Load the manifest.
-  player.load(asset.manifestUri).then(function() {
+  var configureCertificate = Promise.resolve();
+
+  if (asset.certificateUri) {
```
Comment (TheModMaker) [inline demo/asset_section.js]: If you're going to have extra fields on the assets, you should add this field to the `shakaAssets.AssetInfo` type definition in `demo/common/assets.js`.

Comment (chrisfillmore) [reply demo/asset_section.js]: Updated.

**Comment by joeyparrish** [comment]
Cherry-picked for v2.2.7.  Thanks!

**Pull request merged**